### PR TITLE
EGRC-469

### PR DESCRIPTION
**Pull request by mikeisen1**
Finished subtask EGRC-469 where I migrated the data into one directory and updated test
and storybook file imports as necessary.

### CODE REVIEW

**Comment by kylelaker** [comment]
Congratulations on getting your first PR opened in the `oscal-react-library` repo! I have re-targeted this PR to have the base branch (the branch that the PR will be merged in to) to the `EGRC-275` branch instead of `develop`. You won't need to change anything locally; this was purely a change on the GitHub side.

There are also a few things that I'd like to make sure we get cleaned up in your local git configuration including the `user.name` and `user.email` values and making sure that they're setup correctly for your repository. We can talk about a few other `git`-related things at the same time.

Love that we are able to move all this stuff into files that can be referenced from various places and deduplicate a lot of this. While everything here look good at first glance, I am going to hold off on approving until I've had a chance to look a little more closely and we've been able to work together on some of the `git` configuration items.

**Comment by tuckerzp** [comment]
For the linting errors, I believe that @hreineck has had a similar issue in the past. I think the fix was to make sure you update and commit `example/package-lock.json` & `package-lock.json`. It is running `npm install` and updating those files since they are not up to date.

**Comment by kylelaker** [comment]
@tuckerzp Good catch! That is #135 as well.

File: src/test-data/BackMatterData.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        },
+      ],
+    },
+    {
+      uuid: "dc380596-027f-423b-83f2-82757554ee28",
+      title: "Resource Test Title 2",
+      description: "This is a test description for resource 2",
+      rlinks: [
+        {
+          href: href,
```
Comment (hreineck) [inline src/test-data/BackMatterData.js:32]:
```suggestion
          href,
```

When I run `npm run storybook` I get a linter issue here. This should fix it.

File: src/stories/OSCALModification.stories.js [hunk 2]
```diff
@@ -10,26 +11,7 @@ const Template = (args) => <OSCALControlModification {...args} />;
...
-        },
-      ],
-    },
-  ],
-};
-
 Default.args = {
-  modifications: exampleModifications,
+  modifications: exampleModificationsAddsTwo,
   controlId: "control-1",
```
Comment (hreineck) [inline src/stories/OSCALModification.stories.js:16]:
```suggestion
  controlId: "control-1",
  controlPartId: "control-1_smt.a",
```

Tested this with `npm run storybook` - Without this `controlPartId` prop the icon will be invisible.

**Review by hreineck — CHANGES_REQUESTED** [review]
Awesome work, Michael! Have a few small issues to resolve but this is really great overall.

**Comment by mikeisen1** [comment]
I committed the changes suggested by @hreineck and updated the package-lock.json, but I am still getting the issues of failing the linter check.

**Comment by hreineck** [comment]
> I committed the changes suggested by @hreineck and updated the package-lock.json, but I am still getting the issues of failing the linter check.

Try `npm install` and then committing package-lock.json

**Comment by hreineck** [comment]
Actually, looks like it's complaining about example/package-lock.json as well. Maybe try committing the changes to that too.

**Comment by tuckerzp** [comment]
> Try `npm install` and then committing package-lock.json

I am surprised that [fff7cb9](https://github.com/EasyDynamics/oscal-react-library/pull/132/commits/fff7cb96cf9514948e56dd23910f09107c1d1792) doesn't fix that.

**Comment by mikeisen1** [comment]
For the lint checks, I was initially having an issue of not being able to checkout another branch even locally. I fixed that issue, ran the npm install command in both directories,  and double-checked that the git checkout to another branch would work. That is now working locally but the lint check is still failing. Does anyone have advice on how I can resolve this?

**Review by hreineck — APPROVED** [review]
Awesome job!

**Comment by kylelaker** [comment]
@mikeisen1 now that we've merged all the other stuff into EGRC-275, can you take one more look at everything to make sure all the data is organized correctly? I think #116 may have added more stuff

**Review by kkennedy26 — APPROVED** [review]
Looks good! Ran the tests and storybook, everything is functioning as expected

**Review by kylelaker — APPROVED** [review]

**Pull request merged**